src/NzbDrone.Common/Http/HttpRequest.cs

```diff
@@ -16,6 +16,8 @@ public HttpRequest(string url, HttpAccept httpAccept = null)
             _segments = new Dictionary<string, string>();
             AllowAutoRedirect = true;
             Cookies = new Dictionary<string, string>();
+            IsIndexerRequest = false;
```

Comment:
HttpRequest should not know about something that uses it, `IsIndexerRequest` should be `UseProxyForRequest` (or something more generic).